### PR TITLE
Fetch country profile data

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses #41 #75 

## Changes

- Fetched country details data, total cases, score card data

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/components/IndicatorChart/styles.css [hunk 1]
```diff
@@ -1,8 +1,7 @@
 .indicator-chart-card {
     display: flex;
     flex-shrink: 0;
-    border: var(--width-separator-thin) solid var(--color-separator-light);
-    border-radius: var(--dui-border-radius-card);
+    border-radius: 0;
```
Comment (subinasr) [inline app/components/IndicatorChart/styles.css]: Please add a note here why you had to override the card style.

File: app/views/Dashboard/Country/index.tsx [hunk 2]
```diff
@@ -16,47 +16,186 @@ import {
...
+`;
+
+const LINE_COLORS = ['#FFDD98', '#ACA28E'];
+
 function Country(props: CountryProps) {
+    const {
+        data: CountryResponse,
+    } = useQuery<CountryQuery>(
+        COUNTRY_PROFILE,
+        { variables: { iso3: 'IND', contextIndicatorsIds: 'total_cases', emergencies: [] } },
```
Comment (subinasr) [inline app/views/Dashboard/Country/index.tsx]:
```suggestion
        { 
            variables: { 
                iso3: 'IND',
                contextIndicatorsIds: 'total_cases',
                emergencies: [],
            },
        },
```

File: app/views/Dashboard/Country/index.tsx [hunk 3]
```diff
@@ -16,47 +16,186 @@ import {
...
+            (acc, item) => ({
+                ...acc,
+                [item.emergency]: item.contextIndicatorValue,
+            }),
+            { date: key },
+        ),
+    );
+
+    const outbreaks = CountryResponse?.countryEmergencyProfile.map((item) => {
+        const colors = LINE_COLORS[Number(item.contextIndicatorValue) % LINE_COLORS.length];
```
Comment (subinasr) [inline app/views/Dashboard/Country/index.tsx:161]: Why?

**Review by subinasr — APPROVED** [review]

File: app/components/IndicatorChart/styles.css (hunk 1 above)
Comment (shreeyash07) [reply app/components/IndicatorChart/styles.css]: Done

**Pull request merged**